scripts/integration/pulsar/test.yaml

```diff
@@ -1,7 +1,7 @@
 features:
 - pulsar-integration-tests
 
-test_filter: '::pulsar::'
+test_filter: '::pulsar::integration_tests::'
```

Comment:
It's come up a few times now- I was chatting with Jesse about that a while back and the consensus we came to is that it's unnecessary to cover unit tests in the integration tests. If the unit tests fail, the CI check will detect that.
It can cause extra noise and extra resources in CI to run the unit tests there. Noise in the case that unit test failure causes the integration test to fail.

Reply:
We can definitely bring that up to discuss with the broader team. I had a TODO at one point to go through all the integration tests and restrict the scope. That was around the time vdev was being created so I put that on hold and didn't come back to it yet.